### ISSUE TITLE
Help with maintainance

### ISSUE THREAD

**Issue by addyosmani** [yeoman/grunt-usemin#313]
If you heavily use or have used `grunt-usemin`, we would like to ask if you might be interested in helping us maintain this task. As you can probably tell by the PR queue and questions about the project, it's used by many but needs active weekly review of threads and code we might like to land back to the project.

The rest of the Yeoman team are currently focusing on generators, docs and our next version of the project so any help at all would be hugely appreciated.

**Comment by arthurvr** [comment]
I just saw you guys deprecated this repo, tho a note in the repo description isn't enough I think. I'd prefer to have, at least, little of a note in the readme where we also suggest alternatives.

**Comment by arthurvr** [comment]
I'm def. interested in trying to help out, btw.

**Comment by jfroffice** [comment]
:+1: if I can help to review some PR or fix some issues.

**Comment by nodox** [comment]
I'm always looking for ways to contribute on Github. Would love to help in anyway possible!

**Comment by shamoons** [comment]
I'm happy to take on as maintainer.

**Comment by karensg** [comment]
That would be great! Lots of pull requests are waiting review :)

**Comment by nodox** [comment]
Would love to take on as another maintainer.

**Comment by ptitgraig** [comment]
@addyosmani  @sindresorhus,  some people seems to be keen on maintaing this project :)

**Comment by sindresorhus** [comment]
@shamoons @karensg @nodox @ptitgraig Thanks for stepping up. You're now collaborators on this repo :)

The guidelines are simple. Be kind to everyone. Larger changes should come in a pull request. Smaller ones can be committed directly. Pull requests should have at least one :+1: from another collaborator before being merged. Follow the existing code style and ensure `npm test` passes before pushing a commit. You can use branches in this repo, but clean them up afterwards. Bugs fixes and new features should come with tests. We prefer a clean commit history, so when merging a pull request, either ask the contributor to [squash](http://stackoverflow.com/questions/11396069/squash-to-only-one-proper-commit-for-github-pull-request) their commits or do it for them manually.

Also keep in mind that the goal of this project is simplicity. That means solving a problem for the 90% with the least amount of options.

The next step and biggest priority is to land @stephanebachelier's [dev branch](https://github.com/yeoman/grunt-usemin/tree/dev) which resolves https://github.com/yeoman/grunt-usemin/issues/244.

**Comment by eddiemonge** [comment]
Or make this a thin wrapper around useref and combine the efforts with @jonkemp to reduce duplication

**Comment by nodox** [comment]
What needs to happen to @stephanebachelier [dev branch](https://github.com/yeoman/grunt-usemin/tree/dev) before it can be merged? Are we missing tests?

**Comment by sindresorhus** [comment]
@nodox I don't know. You'll have to ask him.

**Comment by eddiemonge** [comment]
Someone just needs to go through it and make sure it works and merges cleanly

**Comment by stephanebachelier** [comment]
@sindresorhus @eddiemonge I'm back in the place. For the dev branch I had asked if anybody could tested it but I don't know about anyone apart from me.
I will take a look again at the dev branch. It should still works.

Is there some forum / place to discuss about what to do ? I created the gitter yeoman/grunt-usemin channel last year but there is not so much activity :) Have you got another channel ?

**Comment by sindresorhus** [comment]
Yes, you can use https://gitter.im/yeoman/yeoman (Maybe add that to the readme).

**Comment by stephanebachelier** [comment]
@sindresorhus ok done.

**Comment by benmcginnis** [comment]
still need any help @addyosmani and @sindresorhus ?

**Comment by sovaj** [comment]
I like to help you guys

Let me know if you still looking for someone to maintain this project. If you do I will gladly do it.

Thanks.

**Comment by eddiemonge** [comment]
Yup still are. What capacity would you like to help in?
